presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -335,6 +335,10 @@ Property Name                                      Description
 ``hive.metastore-refresh-max-threads``  Maximum threads used to refresh cached metastore data.        100
 
 ``hive.metastore-timeout``              Timeout for Hive metastore requests.                         ``10s``
+
+``hive.hide-delta-lake-tables``         Controls whether to hide Delta Lake tables in table          false
+                                        listings. Currently applies only when using the AWS Glue
```

Comment:
We should say "when using the AWS Glue metastore" since to better match the existing language

Reply:
Fixed. Added to the same commit with --append.